okrs.md

```diff
@@ -0,0 +1,43 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### Implement updated IPLD Format spec
+
+This relates to https://github.com/ipld/interface-ipld-format/pull/22
+
+ - `P0` - @stebalien (?) finish deciding on the primitives: https://github.com/ipld/specs/issues/56
+ - `PX` - @vmx - Finish updating the IPLD format spec
+ - `PX` - @vmx - Have proper JSON serialization for JavaScript formats (Bitcoin, CBOR, DAG-PB, Ethereum, Git, Zcash)
+ - `PX` - OWNER - Have proper JSON serialization for Go formats (Bitcoin, CBOR, DAG-PB, Ethereum, Git, Zcash)
```

Comment:
What does "proper JSON serialization" mean?

Reply:
I mean a JSON representation that is agreed on between the JS and Go implementations. I'm happy to change to some better wording.

Reply:
Ah. You're not talking about the JSON IPLD *format* but the representation (e.g., in the API).

Reply:
The toJSON method